### DISCUSSION TITLE
Anyone incorporating known camera poses?

### DISCUSSION THREAD

**Discussion by claytonrothschild** [OpenStitching/stitching#248] [General]
Has anyone incorporated known camera parameters (intrinsics/extrinsics) into the stitching solution? I have done a planar stitch using simple 2d homography but have challenged warping... must do feature mapping but feature mapping is not consistent enough to be without error. Thanks.

**Comment by lukasalexanderweber** [comment]
No till now there is only this Idea: https://github.com/OpenStitching/stitching/discussions/174
